models/intermediate/field_history/int_jira__issue_calendar_spine.sql

```diff
@@ -1,14 +1,15 @@
 with spine as (
 
+    -- depends_on: {{ ref('stg_jira__issue') }}
     {% if execute and flags.WHICH in ('run', 'build') %}
     {% set first_date_query %}
     -- start at the first created issue
-        select  
+        select
             coalesce(
-                min(cast(created as date)),
+                min(cast(created_at as date)),
                 cast({{ dbt.dateadd("month", -1, "current_date") }} as date)
             ) as min_date
-        from {{ source('jira','issue') }}
+        from {{ ref('stg_jira__issue') }}
```

Comment:
Catherine and I were thinking it would be most seamless to switch to referencing the _tmp model, since this is basically just the unioned source tables. Could you switch to referencing `stg_jira__issue_tmp` instead of `stg_jira__issue`? This also means we can revert the line 9 change

Reply:
Updated